python/pyspark/streaming/tests.py

```diff
@@ -1118,4 +1210,4 @@ def search_kinesis_asl_assembly_jar():
                                mqtt_assembly_jar, mqtt_test_jar)
 
     os.environ["PYSPARK_SUBMIT_ARGS"] = "--jars %s pyspark-shell" % jars
-    unittest.main()
+    unittest.main(verbosity=3)
```

Comment:
It is useful for debugging in the generated log file python/unit_tests.log. Without this, the log file only shows which tests have finished, not which ones have started.